docker/README.md

```diff
@@ -0,0 +1,37 @@
+Sawtooth Lake - Docker
```

Comment:
This should be in the Sawtooth 0.8 documentation somewhere, not a README, perhaps as part of a contributors guide.